src/hooks/useParticipationAvatars.ts

```diff
@@ -0,0 +1,91 @@
+import { Participation } from "../generated"
+import { useCurrentGame } from "./api/useCurrentGame"
+
+const shortHandLength = 2
+
+export type AvatarConfig = {
+  shortHand: string
+  hue: number
+}
+
+export function useParticipationAvatars() {
+  const { game } = useCurrentGame()
```

Comment:
Das problem war, dass ich hier die activeParticipations verwendet habe.
Das heisst eine nicht active (nicht turnActive, aber active=false) hatte kein ShortHand -> error.

Ich weiss aber nicht, ob jemand von euch das Spiel verlassen hat, bevor wir den ersten whitescreen hatten.
Ist aber der einzige weg, ich ich das bis jetzt reproduzieren konnte.

Reply:
I think Carmen left the game when it crashed for us. We only managed to reproduce it like that anyways. So this works for me!